docs/content/component-data.md

```diff
@@ -9,56 +9,33 @@ SpatialOS components are generated from `.schema` files into components that the
 ### Overview
 
 A `struct`, which implements `Unity.Entities.IComponentData` and `Improbable.Gdk.Core.ISpatialComponentData`,
-is generated for each [blittable](https://docs.microsoft.com/en-us/dotnet/framework/interop/blittable-and-non-blittable-types) SpatialOS component.
-A `class`, which extends `UnityEngine.Component` and implements `Improbable.Gdk.Core.ISpatialComponentData` is generated for each non-blittable SpatialOS component.
+is generated for each SpatialOS component.
 
 Each of these structs will be named `SpatialOS[schemalang component name]` and will contain only the schema data fields.
```

Comment:
This sentence doesn't make it clear who does the naming. Is it the generation process?
Also "will be named"  and "will contain" reads like a prophecy rather than a statement.  
Is "[schemalang component name]" dependent on the component?
Suggest:

" The generation process names each of these structs according to the relevant schemalang component name,` `SpatialOS[schemalang component name]`. The structs only contain the schema data fields."